src/panels/config/automation/trigger/types/ha-automation-trigger-webhook.ts

```diff
@@ -1,6 +1,10 @@
-import "@polymer/paper-input/paper-input";
+import "../../../../../components/ha-icon-button";
+import { mdiContentCopy } from "@mdi/js";
 import { html, LitElement } from "lit";
 import { customElement, property } from "lit/decorators";
+import { PaperInputElement } from "@polymer/paper-input/paper-input";
```

Comment:
```suggestion
import type { PaperInputElement } from "@polymer/paper-input/paper-input";
```

Reply:
Done in [6721694](https://github.com/home-assistant/frontend/pull/11568/commits/67216944d2752b622c82f29224339800d16e9a87).